toolchain/check/diagnostic_emitter.h

```diff
@@ -51,14 +51,20 @@ class DiagnosticEmitter : public DiagnosticEmitterBase {
   auto ConvertLocImpl(SemIR::LocId loc_id, bool is_token_only,
                       ContextFnT context_fn) const -> Diagnostics::ConvertedLoc;
 
-  // Converts a node_id corresponding to a specific sem_ir to a diagnostic
-  // location.
+  // Converts an `absolute_node_id` import from C++ or corresponding to a
+  // specific sem_ir to a diagnostic location.
   auto ConvertLocInFile(SemIR::AbsoluteNodeId absolute_node_id, bool token_only,
                         ContextFnT context_fn) const
       -> Diagnostics::ConvertedLoc;
 
+  // Converts a `node_id` corresponding to a specific sem_ir to a diagnostic
+  // location. `check_ir_id` must not be `Cpp`.
```

Comment:
I'm not sure this needs to be mentioned here -- C++ files don't have Parse::NodeIds.
```suggestion
  // location.
```

Reply:
Makes sense. Done.